gcc/rust/hir/rust-hir-dump.cc

```diff
@@ -264,8 +323,22 @@ void
 Dump::visit (UseDeclaration &)
 {}
 void
-Dump::visit (Function &)
-{}
+Dump::visit (Function &function)
+{
+  indent++;
+  stream << std::string (indent, indent_char);
+  stream << "function"
+	 << " "
+	 << ":";
+  stream << "[" << std::endl;
```

Comment:
```suggestion
  stream << "Function"
	 << " "
  stream << "{" << std::endl;
```  
Since you're printing an object, shouldn't this use curly brackets instead of square brackets?